README.md

```diff
@@ -5,8 +5,6 @@
 <p align="center">
 <a href="https://github.com/open-horizon/open-horizon.github.io" alt="GitHub contributors">
 <img src="https://img.shields.io/github/contributors/open-horizon/open-horizon.github.io.svg"/></a>
-<a href="https://github.com/open-horizon/open-horizon.github.io" alt="anazdocscopy.yml">
-<img src="https://github.com/open-horizon/open-horizon.github.io/actions/workflows/anaxdocscopy.yml/badge.svg"/></a>
```

Comment:
Everything else LGTM, but we can skip this change as it is already addressed in PR https://github.com/open-horizon/open-horizon.github.io/pull/405, merging that PR before this PR might cause merge conflict, we can either skip this change in this PR or we can just close the previous PR i.e (https://github.com/open-horizon/open-horizon.github.io/pull/405) and link both issues to this single PR.

Reply:
Let's just merge this one.  I didn't rebased my local clone.  Sweep both in.   I'll close #405